_docker/lib/export/drupal_page_url_list_generator.rb

```diff
@@ -42,6 +42,7 @@ def parse_sitemap(sitemap_contents)
       links << link.content
     end
 
+    links.uniq!
```

Comment:
We seem to have duplicate links in the sitemap.xml for some reason (particularly relating to quickstarts)